include/Drivers/MotorController.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef MOTORCONTROLLER_HPP
+#define MOTORCONTROLLER_HPP
+
+#include <Arduino.h>
+#include "RoboClaw.h"
+
+namespace motors {
+
+enum MotorMode { TORQUE, SPEED, VELOCITY };
```

Comment:
I can't find any place where this is being used. Why is it here?

Reply:
Not used, just deleted